### DISCUSSION TITLE
eIDAS compliance

### DISCUSSION THREAD

**Discussion by ayoahha** [OpenSignLabs/OpenSign#178] [General]
Hello there
I just saw that on your FAQ you are mentionning :

> OpenSign™ is designed to comply with the highest standards of digital signature technology, and while it facilitates the creation of digital signatures, the legal compliance in any given jurisdiction depends on the use case and adherence to local laws. OpenSign™ encourages users to consult with legal experts to ensure compliance with regional legislation.

Did you receive any "certification" to jutify eIDAS compliance ?
I mean it seems important to be certified (independent certification) against European Commission-recommended technology standards to justify the compliance...
Is there any plan to so ?

Many thanks.
Keep up the good work 👍

**Comment by andrew-opensignlabs** [comment]
Currently the signatures generated by OpenSign are compliant with eIDAS simple signatures. We will be adding full support for Advanced & qualified signatures in future. We may be doing certain 3rd party tie-ups to comply with various regional laws.

**Reply by ovizii** [reply]
This is exactly what I was looking for. Maybe you should explain this somewhere in slightly more detail for people looking for this info.

![image](https://github.com/OpenSignLabs/OpenSign/assets/837902/8b6d190c-4e45-4217-aa51-82006dc583b6)

**Reply by lmtr0** [reply]
@andrew-opensignlabs Any updates?
seeing as this is 2 years old now, I assume there are new things coming

**Comment by Fangh** [comment]
I tried to upload a file to french a governement agency : INPI and they say that OpenSign doesn't work :

> Votre signature n'est pas valide : La synthèse a été signée avec un certificat étranger hors UE et dont l'autorité de certification n'est pas présente parmi celles définies dans les listes de confiance eIDAS. Votre signature n'est donc pas considérée comme valide puisque nous ne retrouvons pas l'autorité de certification du certificat utilisé parmi les listes de confiance eIDAS. Nous vous invitons à vous rapprocher de votre fournisseur de signature afin d'avoir une signature conforme.

**Reply by metaphys** [reply]
This only says that the certificate is not valid, did you had a valid EU EIDAS certificate (ie not self generated?)

**Reply by Fangh** [reply]
No I did not. I don't know how to get one.

**Reply by metaphys** [reply]
In France ANSSI is listing the authorized sellers:
[Obtenir un certificat de signature électronique](https://cyber.gouv.fr/reglementation/reglementation-identite-confiance-numerique/securite-echanges-voie-electronique/reglement-eidas/obtenir-certificat-signature-electronique/)

BUT I’m not sure that would work with opensign because of  https://github.com/vbuch/node-signpdf/issues/131
I don’t know what is the cryptographic format of eidas certificate, if it’s RSA based it should work but if it’s ECDSA it won’t
